lib/actions/device.coffee

```diff
@@ -300,18 +293,18 @@ exports.init =
 
 			(callback) ->
 				console.info("Checking application: #{options.application}")
-				resin.models.application.get(options.application, callback)
+				resin.models.application.get(options.application).nodeify(callback)
 
 			(application, callback) ->
 				async.parallel
 
 					manifest: (callback) ->
 						console.info('Getting device manifest for the application')
-						resin.models.device.getManifestBySlug(application.device_type, callback)
+						resin.models.device.getManifestBySlug(application.device_type).nodeify(callback)
 
 					config: (callback) ->
 						console.info('Fetching application configuration')
-						resin.models.application.getConfiguration(options.application, networkOptions, callback)
+						resin.models.application.getConfiguration(options.application, networkOptions).nodeify(callback)
```

Comment:
This function doesn't exist in the SDK anymore. Can you install and reuse https://github.com/resin-io/resin-device-config instead here?